### PR TITLE
*: Cut v1.5.0-alpha.0

### PR DESCRIPTION
**Pull request by mxinden**
**What this PR does / why we need it**:

Cut a first v1.5.0 alpha release.

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/kube-state-metrics/issues/557

### CODE REVIEW

**Comment by mxinden** [comment]
Based on the Makefile I am guessing the version is determined based on git tags @brancz?

**Comment by brancz** [comment]
@mxinden that's correct.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
-## Unreleased
-
-- [CHANGE] Disable gzip compression of kube-state-metrics responses by default. Can be re-enabled via `--enable-gzip-encoding`. See #563 for more details.
+## v1.5.0 / 2018-11-30
```
Comment (brancz) [inline CHANGELOG.md]: That would be `v1.5.0-alpha.0`. We typically included all pre-releases' history in Kubernetes project releases.

**Comment by mxinden** [comment]
Thanks for the catch. Would you mind taking another look @brancz?

Can you cut the release-0.15 branch afterwards?

**Comment by brancz** [comment]
This is a huge step for scalability of this project, thank you for all your work @mxinden!

/lgtm

@piosz @loburm @kawych Once I tag and push the image can you make sure to publish the image on gcr.io please? Thanks!

**Comment by brancz** [comment]
The container image for the alpha is available under `quay.io/coreos/kube-state-metrics:v1.5.0-alpha.0`.

**Pull request merged**